### PR TITLE
Update README on how to fix Android keyboard

### PR DESCRIPTION
**Pull request by chandlervdw**
I found this in another issue and I think it should be updated here. https://github.com/FaridSafi/react-native-gifted-chat/issues/680#issuecomment-377662861

### CODE REVIEW

**Comment by khanguslee** [comment]
I think you're missing the expression to return if your condition is false in your ternary operator.

Something like below would work instead?
```
   {Platform.OS === 'android' ? <KeyboardAvoidingView behavior="padding" /> : null}
```

File: README.md [hunk 1]
```diff
@@ -382,16 +382,11 @@ If you are using Create React Native App / Expo, no Android specific installatio
...
 <View style={{ flex: 1 }}>
-   {
-      Platform.OS === 'android' ?
-         <KeyboardAvoidingView behavior="padding">
-            <GiftedChat />
-         </KeyboardAvoidingView> :
-         <GiftedChat />
-   }
+   <GiftedChat />
+   {Platform.OS === 'android' ? <KeyboardAvoidingView behavior="padding" />}
```
Comment (khanguslee) [inline README.md]: I think you're missing the expression to return if your condition is false in your ternary operator.

Something like below would work instead?

   {Platform.OS === 'android' ? <KeyboardAvoidingView behavior="padding" /> : null}

Comment (chandlervdw) [reply README.md]: d'oh! Thanks.

**Comment by chandlervdw** [comment]
@khanguslee thanks for the feedback. Ternary fixed.

**Pull request closed without merging**